plugins/ros-pac-man/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": "1",
+    "information": {
+        "name": "pacman",
```

Comment:
name of the plugin in manifest should correspond to the directory here, can you rename it?
I think ideal name would be `routeros-packet-manager` for both directory and manifest

Reply:
Edited to reflect new package name.